zerver/lib/actions.py

```diff
@@ -4016,16 +4021,26 @@ def do_invite_users(user_profile: UserProfile,
             errors.append((email, email_error))
         elif email_skipped:
             skipped.append((email, email_skipped))
+            if 'deactivated' in email_skipped:
+                deactivated += 1
+
+    activation_msg = ''
+    if deactivated:
+        if user_profile.is_realm_admin:
+            activation_msg = "You can reactivate a deactivated user in organization settings."
```

Comment:
These should be marked for translation

Reply:
@rishig Can you please tell what is translation, and how is it done?

Reply:
Okay I think so I got this. `_()` in the django and `i18n.t` in javascript. Is that correct?